src/Rings/MPolyMap/MPolyQuo.jl

```diff
@@ -4,31 +4,39 @@
 #
 ################################################################################
 
-function _check_imgs_quo(R, S::NCRing, imgs)
+function _check_imgs_quo(R, S::NCRing, imgs, coeff_map = nothing)
   n = length(imgs)
   for i in 1:n, j in 1:(i - 1)
     @req imgs[i] * imgs[j] == imgs[j] * imgs[i] "Images $i and $j do not commute"
```

Comment:
Is there a special case for commutative rings?
If the images are complicated this could be expensive.